angular/test/apps/ng14/src/app/version-test/version-test-routing.module.ts

```diff
@@ -3,12 +3,7 @@ import { RouterModule } from "@angular/router";
 
 @NgModule({
   imports: [
-    RouterModule.forChild([
-      {
-        path: 'standalone',
```

Comment:
Do we still need standalone component tests? My understanding is we're not removing support, we're just removing the old environment injector prop

Reply:
Standalone routing is now supported for all versions of Angular that we test. I just relocated the test (and updated the path) from being in each individual test app, to being in the "base" test app that all the test apps use.